Makefile

```diff
@@ -0,0 +1,13 @@
+up:
+	@docker-compose -f local.yml up
+
+build:
+	@docker-compose -f local.yml up --force-recreate --build
+
+manage:
+	@docker-compose -f local.yml run --rm django python ./manage.py $(filter-out $@,$(MAKECMDGOALS)) --settings=config.settings.local
```

Comment:
I use a `Makefile` for docker too, and IMHO its quite useful because you can easily change from local to production;
```
buildev:
@sed -i.bak s/DJANGO_SETTINGS_MODULE=config.settings.production/DJANGO_SETTINGS_MODULE=config.settings.local/g .env
	@sed -i.bak s/DJANGO_DEBUG=False/DJANGO_DEBUG=True/g .env
	@docker-compose -f docker-compose-dev.yml up --build
```
and later on:
```
buildprod:
	@sed -i.bak s/DJANGO_SETTINGS_MODULE=config.settings.local/DJANGO_SETTINGS_MODULE=config.settings.production/g .env
	@sed -i.bak s/DJANGO_DEBUG=True/DJANGO_DEBUG=False/g .env
	@docker-compose -f docker-compose.yml up --build
```
this way its idempotent. It doesnt matter if you are already on `dev` environment, it will work as expected and it will change seamlessly to production in one command.